src/ConnectedMode/UI/Resources/Styles.xaml

```diff
@@ -3,9 +3,31 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
              xmlns:platformUi="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0">
+
+    <SolidColorBrush x:Key="Border" Color="#FFD5DFE5"/>
```

Comment:
hardcoded color

Reply:
It's not theme dependant. Choose one of the VS colors

Reply:
It looks good on all themes